src/App.js

```diff
@@ -1,29 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
```

Comment:
- [ ] I think you shouldn't have deleted this `App.js` file. It is a common practice to use the `App` component as the top-level wrapper of components. Kindly revert this change and import and render the `Calculator` component in the `App` component. :pray: 
- [ ] Also, kindly refactor it to a React class component as you are required to practice React components in this milestone.